lib/vsc/utils/rest.py

```diff
@@ -111,47 +111,47 @@ def get(self, url, headers=None, **params):
         Do a http get request on the given url with given headers and parameters
         Parameters is a dictionary that will will be urlencoded
```

Comment:
add that the url itself cannot be urlencoded

Reply:
so this PR breaks all code that passes already encoded urls (to address this issue most likely)